sample-apps/js-sdk-aws-XRay/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:14
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm install
+
+COPY . .
+
+EXPOSE 8080
```

Comment:
you might not need this, since you will need to read the env var LISTEN_ADDRESS

Reply:
removed